t/build_man.t

```diff
@@ -37,8 +37,6 @@ END {
 
 ok( chdir 'Big-Dummy', "chdir'd to Big-Dummy" ) ||
   diag("chdir failed: $!");
-my $README = 'README.pod';
-{ open my $fh, '>', $README or die "$README: $!"; }
```

Comment:
The warnings this causes are not unique to Windows.  The tests later in the file are meant to validate that no extra man page files get generated, including making sure it doesn't generate them for a top level README.pod file.  Possibly the warnings should be suppressed rather than removing this.

Reply:
Hmm, that's what I proposed initially with https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/commit/1ebf6e6293cb842ee651d597ba82e621b3f36098 but @karenetheridge and @Grinnz objected in the conversation above... Is https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/commit/1ebf6e6293cb842ee651d597ba82e621b3f36098 how you think it should be resolved? Or do you have another suggestion?

Reply:
My suggestion was not to touch this at all in this PR since it's unrelated.

Reply:
I see this PR as "add MSWin32 testing" and "fix the problems in that testing environment". This cleans up the testing, so, IMO, it's related.

Since the change is in a single commit, it could be be easily moved to another PR. But, for the same reason, including it here is also easy and fixes the distracting warnings.

Is the point that you want to debate whether this should be done or how it should be done in another thread?

Reply:
> Is the point that you want to debate whether this should be done or how it should be done in another thread?

I think given the discussion above, that is a good idea. If it takes >1 PR to get Windows in the needed place, so be it.

Reply:
* commit removed